packages/0x.js/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
     "extends": "../../tsconfig",
     "compilerOptions": {
-        "outDir": "lib"
+        "outDir": "lib",
+        "rootDir": "."
```

Comment:
`rootDir` is required here because tslint breaks if it is missing and tsconfig.json contains `composite: true`. `composite: true` is required in order to enable project references.